bin/commands/init/generate/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+// This program and the accompanying materials are made available
+// under the terms of the Eclipse Public License v2.0 which
+// accompanies this distribution, and is available at
+// https://www.eclipse.org/legal/epl-v20.html
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as os from "cm_os";
+import * as xplatform from "xplatform";
+import * as fs from '../../../libs/fs';
+import * as config from '../../../libs/config';
+import * as common from '../../../libs/common';
+import * as zosFs from '../../../libs/zos-fs';
+import * as zosJes from '../../../libs/zos-jes';
+
+export function execute(dryRun?: boolean) {
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+  const tempFile = fs.createTmpFile();
+  zosFs.copyMvsToUss(ZOWE_CONFIG.zowe.setup.dataset.prefix + '.SZWESAMP(ZWEGENER)', tempFile);
+  let jclContents = xplatform.loadFileUTF8(tempFile, xplatform.AUTO_DETECT);
+
+  jclContents = jclContents.replace(/\{zowe\.setup\.dataset\.prefix\}/gi, ZOWE_CONFIG.zowe.setup.dataset.prefix);
```

Comment:
1) Schema valiadation for `zowe.setup.dataset.prefix` should be limited to `"maxLength": 35`.
`zowe.setup.dataset.prefix` with max lenght and `zwe generate`:
```
...
              ----+----1----+----2----+----3----+----4----+----5---
Template JCL: JOKER01.ZOWE.PR#3718.$$$$$$$$.@###@.#------@.SZWESAMP(ZWEGENER)
...
```

2) Replace catch:

Replace is affected by [special replacement patterns](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_string_as_the_replacement), in this example every `$$` is replaced by `$`:

```
...
Template JCL: JOKER01.ZOWE.PR#3718.$$$$$$$$.@###@.SZWESAMP(ZWEGENER)`
...
//* Replace JOKER01.ZOWE.PR#3718.$$$$.@###@ with the
//* Value as seen in zowe.yaml
//*
//SYSPROC  DD   DSN=JOKER01.ZOWE.PR#3718.$$$$.@###@.SZWEEXEC,DISP=SHR
...
```

Reply:
Schema validation updated.
Replace updated.